README.md

```diff
@@ -1,8 +1,19 @@
 # styles
 The billinge group matplotlib style sheet.
 
-To use this style put the following into your plot generating code:
+## Workflow
+1. clone to local drive via `git clone https://github.com/pdfttp/styles.git`
+1. put the following into your plot generating code:
 ```python
 import matplotlib.pyplot as plt
 plt.style.use(os.path.join('/path/to/this/folder', 'billinge.mplstyle'))
 ```
+
+OR
+
+If you are doing more intense work and are expecting to need to change this please:
```

Comment:
could you mention that styles can be overridden by redefining in the main document, but if you want to edit the style file for reuse in multiple plots then do the copy-paste...  thx